rust/polars-ocaml/src/data_frame.rs

```diff
@@ -339,6 +349,36 @@ fn rust_data_frame_diagonal_concat(
         .to_ocaml(cr)
 }
 
+#[ocaml_interop_export]
+fn rust_data_frame_vstack(
+    cr: &mut &mut OCamlRuntime,
+    data_frame: OCamlRef<DynBox<PolarsDataFrame>>,
+    other: OCamlRef<DynBox<PolarsDataFrame>>,
+) -> OCaml<Result<(), String>> {
+    let Abstract(data_frame) = data_frame.to_rust(cr);
+    let Abstract(other) = other.to_rust(cr);
+    let other = match Rc::try_unwrap(other) {
+        Ok(data_frame) => data_frame.into_inner(),
+        Err(data_frame) => data_frame.borrow().clone(),
+    };
+    let result = data_frame
+        .borrow_mut()
+        .vstack_mut(&other)
+        .map(|_| ())
+        .map_err(|err| err.to_string());
+    result.to_ocaml(cr)
```

Comment:
let's use the `dyn_box_result!` macro here.

Reply:
done